modules/ROOT/pages/dw-coercions-functions-todate.adoc

```diff
@@ -6,6 +6,54 @@ endif::[]
 
 
 [[todate1]]
+== toDate&#40;str: String, formatters: Array<Formatter&#62;&#41;: Date
+
+Transforms a `String` value into a `Date` value using the first `Formatter` that
+matches with the given value to transform.
+
+_Introduced in DataWeave version 2.5.0._
```

Comment:
It's a bit odd that we are just adding these now.

Reply:
I'll make sure to propagate these changes into 2.5 and 2.6.